auto_submit/test/requests/check_pull_request_test.dart

```diff
@@ -2,47 +2,102 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: constant_identifier_names
+
 import 'package:auto_submit/requests/check_pull_request.dart';
+import 'package:auto_submit/requests/check_pull_request_queries.dart';
 import 'package:github/github.dart';
+import 'package:meta/meta.dart';
 import 'package:shelf/shelf.dart';
 import 'package:test/test.dart';
+import 'package:graphql/client.dart' hide Response;
 
 import './github_webhook_test_data.dart';
 import '../requests/github_webhook_test_data.dart';
 import '../src/request_handling/fake_pubsub.dart';
 import '../src/service/fake_config.dart';
 import '../src/service/fake_github_service.dart';
+import '../src/service/fake_graphql_client.dart';
 
 void main() {
   group('Check CheckPullRequest', () {
     late CheckPullRequest checkPullRequest;
     late FakeConfig config;
+    late FakeGraphQLClient githubGraphQLClient;
     final FakeGithubService githubService = FakeGithubService();
     final FakePubSub pubsub = FakePubSub();
+    late PullRequestHelper prRequested1;
+    late PullRequestHelper prRequested2;
+    late List<QueryOptions> expectedOptions;
+    late QueryOptions flutterOption;
+    late QueryOptions cocoonOption;
     const String testTopic = 'test-topic';
-    const String login = "engine-flutter-autoroll";
+    const String rollorAuthor = "engine-flutter-autoroll";
     const String labelName = "warning: land on red to fix tree breakage";
-    const String repoName = 'cocoon';
-    const String autosubmitLabel = 'no_autosubmit';
+    const String cocoonRepo = 'cocoon';
+    const String noAutosubmitLabel = 'no_autosubmit';
+
+    setUp(() {
+      githubGraphQLClient = FakeGraphQLClient();
+      expectedOptions = <QueryOptions>[];
+
+      githubGraphQLClient.queryResultForOptions = (QueryOptions options) {
+        expect(options.variables['sOwner'], 'flutter');
+        final String? repoName = options.variables['sName'] as String?;
+        if (repoName == 'flutter') {
+          return createQueryResult(prRequested1);
```

Comment:
prRequested1 -> flutterRequest

Reply:
Sure, sorry I'm not good at naming variables.